### PR TITLE
Sql schema and data

### PR DESCRIPTION
**Pull request by Anthony-Cona**
Added schema.sql and data.sql, allows for swagger to still be used and Spring data queries to still be used. Possible Update to remove models, unsure if possible could not find a proper example.

### CODE REVIEW

File: Curricula/src/main/java/com/revature/models/Skill.java [hunk 1]
```diff
@@ -6,12 +6,12 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @Entity
-@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})// put this on all of your entities
```
Comment (Abatson) [inline Curricula/src/main/java/com/revature/models/Skill.java:14]: why was this removed?

File: Curricula/src/main/java/com/revature/models/Category.java [hunk 2]
```diff
@@ -11,7 +11,6 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @Entity
-@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})// put this on all of your entities
```
Comment (Abatson) [inline Curricula/src/main/java/com/revature/models/Category.java:14]: Why was this removed

**Review by Abatson — CHANGES_REQUESTED** [review]
Why remove the jsonignore properties

File: Curricula/src/main/java/com/revature/models/Skill.java (hunk 1 above)
Comment (Anthony-Cona) [reply Curricula/src/main/java/com/revature/models/Skill.java:14]: It was removed because it didnt need to be there. I can add it back in

File: Curricula/src/main/java/com/revature/models/Category.java (hunk 2 above)
Comment (Anthony-Cona) [reply Curricula/src/main/java/com/revature/models/Category.java:14]: It was removed because it didnt need to be there. I can add it back in

**Review by Abatson — APPROVED** [review]
